### PR TITLE
Add option to conda run to not capture the output

### PR DESCRIPTION
**Pull request by xhochy**
This allows the use of interactive commands like ipython.

### CODE REVIEW

**Comment by xhochy** [comment]
Can someone give me a hint where `conda run` is actually tested. I fail to see the correct location.

**Comment by xhochy** [comment]
@conda-core rebased and CI passed, ready for review.

**Comment by chenghlee** [comment]
> Can someone give me a hint where `conda run` is actually tested? I fail to see the correct location.

The `TestRun` class in `tests/test_cli.py`

File: conda/cli/conda_argparse.py [hunk 1]
```diff
@@ -1090,6 +1090,12 @@ def configure_parser_run(sub_parsers):
         help="Current working directory for command to run in.  Defaults to cwd",
         default=os.getcwd()
     )
+    p.add_argument(
+        "--no-capture-output",
+        help="Don't capture stdout/stdout",
```
Comment (dhirschfeld) [inline conda/cli/conda_argparse.py:1095]:
```suggestion
        help="Don't capture stdout/stderr",
```

**Comment by xhochy** [comment]
Thanks @chenghlee & @dhirschfeld. I've added a test that checks that the output is not captured by `conda` but rather passed though. I had to move one testclass to a pytest-Test-class but since the tests are generally called by pytest, I hope this is ok.

**Review by chenghlee — APPROVED** [review]

**Pull request merged**